sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -83,7 +83,7 @@ export class CertificateClient {
 
 // @public
 export interface CertificateClientOptions extends coreHttp.PipelineOptions {
-    serviceVersion?: "7.0" | "7.1" | "7.2";
+    serviceVersion?: "7.0" | "7.1" | "7.2" | string;
```

Comment:
Most other libraries just use a string for custom service version, and the service returns a useful error when you pass an incorrect version anyway. The only question I have is if there's any value in keeping the hardcoded values in this sum type or just using a string. You lose intellisense anyway, so maybe it's better to just make this a string to align with our other libraries?